portal-api/src/v1/users/routes.js

```diff
@@ -186,6 +186,22 @@ module.exports.remove = (req, res, next) => {
   });
 };
 
+const sendSignInLinkEmailAndHandleErrors = (next) => async (error, user) => {
+  if (error) {
+    next(error);
+  } else if (user) {
+    const { status } = await sendSignInLinkEmail(user.email, user.firstname, 'placeholderSignInLink');
+    if (status === 201) {
+      // TODO DTFS2-6680: Add success logic here.
+    } else {
+      // TODO DTFS2-6680: Add failure logic here.
+    }
+  } else {
+    next(usernameOrPasswordIncorrect);
+  }
+};
+module.exports.sendSignInLinkEmailAndHandleErrors = sendSignInLinkEmailAndHandleErrors;
```

Comment:
Do we only export this just so we can test it?

Reply:
@oscar-richardson-softwire and I spoke about this on a call: the answer is yes, and we discussed how we could test it by through `login` by mocking `findByUsername` instead. We're not going to do so in this PR because this code is about to change anyway